### DISCUSSION TITLE
Allow disabling ConsoleTraversalSummary output

### DISCUSSION THREAD

**Discussion by csmulhern** [biomejs/biome#3981] [Ideas]
There's currently no way to disable this output when invoking biome format (other than redirecting stdout):

```
Formatted 12 files in 17ms. No fixes applied.
```

**Comment by ematipico** [comment]
Would you be able to provide a better proposal?

- why would you want to turn it off (use cases)
- how
- how it would affect error cases

**Comment by csmulhern** [comment]
Sure, I can add a little color. My main use case is to call Biome within a broader formatting ecosystem. E.g. I have a multilanguage monorepo and would like to run a format script that formats all modified files across every language, using Biome for the biome-compatible ones.

In such a case, it'd be preferable to run all the formatting child processes in parallel, and control everything written to stdout / stderr from the main script.

I can easily redirect stdout / stderr of the child processes, but I figured this was also a valid use case for a silent mode. Given that Biome allows control of output via --log-level, I expected the output above to also be ommitted when using `--log-level none`, given its diagnostic in nature.

"how it would affect error cases" - I think outputting to stderr when errors occur is fine even in a silent mode, but that diagnostic information should generally be skipped.
